### DISCUSSION TITLE
What does milvus consistency level really do?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18388] [Q&A and General discussion]
What does milvus consistency level really do?
I mean does it only relates to deleting data? 

What if I only insert (and never or really rarely delete data?)

**Comment by martinenkoEduard** [comment]
Is there a difference if I create multiple connections.connect(
  alias="default", 
  host='localhost', 
  port='19530'
)

Does an  alias="default",  mean something to consistency level ?

**Comment by yhmo** [comment]
Not easy to explain, please refer this article: 
https://github.com/milvus-io/milvus/blob/master/docs/developer_guides/how-guarantee-ts-works.md

The "alias" in connect() is the name of a connection, nothing related to consistency level.
